config/helpers.php

```diff
@@ -55,7 +55,7 @@ function collection(string $name)
 /**
  * Checks / returns a CSRF token
  *
- * @param string $check Pass a token here to compare it to the one in the session
+ * @param string|null $check Pass a token here to compare it to the one in the session
  * @return string|bool Either the token or a boolean check result
  */
 function csrf(string $check = null)
```

Comment:
```suggestion
function csrf(?string $check = null)
```

Reply:
You right. I've tested and works same. `func_num_args()` returns still `0` when no argument passed.

Reply:
✅